components/Ui/UiJoinWithCards.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <section class="cds--grid join-with-cards">
```

Comment:
I believe the class `join-with-cards` is no longer needed here.
```suggestion
  <section class="cds--grid">
```